src/carts/CartMapper.js

```diff
@@ -81,7 +81,7 @@ class CartMapper {
             cartEntries = CartMapper._mapCartEntries(magentoCart.totals.items, magentoCart.totals.quote_currency_code);
         }
 
-        let cart = new Cart(cartEntries, id);
+        let cart = new Cart(magentoCart.currency, cartEntries, id);
```

Comment:
this needs to be refactored to include all required fields but we can leave it for now and updated with builders.